.idea/todo_list.iml

```diff
@@ -28,7 +28,16 @@
     </content>
```

Comment:
`.idea` folder should not be part of the repository.
I see that you have it in (.gitignore file](https://github.com/nofaralfasi/todo_list/blob/main/.gitignore),
but there is trick:

If you commit and push file / folder to the git and later add it to the `.gitignore`, the file/folder won't be ignored in future commits and git will track the changes.

See the [stackoverflow issue](https://stackoverflow.com/q/7527982/18171185), and this [answer](https://stackoverflow.com/a/7528016/18171185)